components/elm/src/biogeophys/EnergyFluxType.F90

```diff
@@ -380,8 +392,171 @@ subroutine Restart(this, bounds, ncid, flag)
          long_name='', units='', &
          interpinic_flag='interp', readvar=readvar, data=this%btran2_patch) 
 
+    call restartvar(ncid=ncid, flag=flag, varname='BTRAN_MIN', xtype=ncd_double,  &
```

Comment:
Can you please confirm that adding this new variable to the restart file doesn't break the backward compatibility of existing restart files that can be used as initial conditions?

Reply:
I think it would actually stop the run. I wasn't aware that restart files should allow resuming runs created with previous versions of the code. I can modify the code to make a dummy restart value in case the variable is missing from the restart, but do you happen to know one example of such code elsewhere in ELM, so I can base the changes on what is already being done? Thanks!